benchmark/util/mime-parser.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+const common = require('../common.js');
+
+const inputs = [
```

Comment:
I don't see this being used in this file. Is this correct?

Reply:
Oh, you're right. Sorry, was a leftover 👍